tokio-reactor/src/lib.rs

```diff
@@ -570,6 +570,7 @@ impl Drop for Inner {
 }
 
 impl Direction {
+    #[allow(clippy::trivially_copy_pass_by_ref)]
```

Comment:
Thanks. I'm pretty sure all of these can / should be fixed. We can handle them later. I added a note in #1083.

Reply:
Done in https://github.com/tokio-rs/tokio/pull/1335/commits/614aa54f7df6a3ddb10bf0170c9a80baabf6394d and https://github.com/tokio-rs/tokio/pull/1335/commits/2a053646f90905ae6211751bb17bc00f48118206.